common/v2/components/WalletUnlock/ViewOnly.scss

```diff
@@ -2,7 +2,6 @@
 @import 'common/sass/mixins';
 
 .ViewOnly {
-  width: 425px;
```

Comment:
Free-rider in the PR but a welcome correction to get it centered.
It appears we need to restyle the screen for this flow. 
Created https://app.clubhouse.io/mycrypto/story/4310/fix-viewonly-add-account-styles to continue on this.